src/Data/Profunctor/Kleisli/Linear.hs

```diff
@@ -43,6 +43,10 @@ instance Control.Applicative f => Strong Either Void (Kleisli f) where
   first  (Kleisli f) = Kleisli (either (Data.fmap Left . f) (Control.pure . Right))
   second (Kleisli g) = Kleisli (either (Control.pure . Left) (Data.fmap Right . g))
 
+instance Control.Applicative f => Monoidal (,) () (Kleisli f) where
+  Kleisli f *** Kleisli g = Kleisli $ \(x,y) -> (,) Control.<$> f x Control.<*> g y
+  unit = Kleisli Control.pure
```

Comment:
Perhaps use `-XInstanceSigs` just for readability. It's a bit of a pause figuring out the type in my head. Same for instances below.

Reply:
If we are to add instance signatures, we should go over the whole module. So this would be a separate PR.